tests/integration_test/firebase_auth/firebase_auth_instance_e2e_test.dart

```diff
@@ -840,6 +840,41 @@ void main() {
             fail(e.toString());
           }
         });
+        test(
+            'should not throw error when app is deleted and reinit with same app name',
+            () async {
+          try {
+            final app = await Firebase.initializeApp(
+              name: 'SecondaryApp',
```

Comment:
Can we put 'SecondaryApp' string in a variable to make sure we are reusing the same and prevent typo in the future?